mobile-ui/src/main/java/com/playone/mobile/ui/playone/PlayoneActivity.kt

```diff
@@ -25,15 +28,48 @@ class PlayoneActivity : BaseActivity() {
 
         viewModel = ViewModelProviders.of(this, viewModelFactory)
             .get(LoginViewModel::class.java)
+        viewModel.isProgressing.observe(this, Observer {
```

Comment:
```kotlin
viewModel.apply {
    isProgressing.observe(...)
    // ...
}
```

You might omit `viewModel` all the time and save your time. 🤞

Reply:
Not so save time I think... but I will try